patch.yml

```diff
@@ -13,6 +13,15 @@
 # limitations under the License.
 
 ---
+- name: Set correct RDBMS release 
+  hosts: localhost
+  gather_facts: false
+  tasks:
+  - include_role:
+      name: patch
+      tasks_from: patch_vars.yml
+  tags: populate_variables
```

Comment:
Call it patch_vars for consistency maybe?  I'll have a related comment inside patch_vars.yml

Reply:
Done